composer.json

```diff
@@ -22,21 +27,21 @@
     },
 
     "require" : {
-        "php"                     : "^7.0 || ^8.0",
+        "php"                     : "^7.1 || ^8.1",
```

Comment:
I guess we can use ^7.4

Reply:
To support maximum compatibility I used lowest possible considering packages update. Please, don't look into symfony/cache 🤣